src/nvim/popupmnu.c

```diff
@@ -219,7 +220,9 @@ void pum_display(pumitem_T *array, int size, int selected)
     if (curwin->w_p_rl) {
       pum_width = pum_col - pum_scrollbar + 1;
     } else {
-      pum_width = Columns - pum_col - pum_scrollbar;
+      assert(Columns - pum_col - pum_scrollbar >= INT_MIN
```

Comment:
There are a lot of asserts to verify parameters are in range. Perhaps we could using something like this?

```c
#include <unistd.h>
#include <assert.h>
#include <limits.h>

#define RANGE(n, min, max) ((n) >= (min) && (n) <= (max))

int main(int argc, char **argv)
{
    assert(RANGE(5u - 10, -100, 0));
    return 0;
}

```